src/actions/index.js

```diff
@@ -0,0 +1,35 @@
+import axios from 'axios';
+
+const loading = username=> ({ type: 'LOADING', payload: username });
+
+const loadResult = (repos) => ({ 
+    type: 'LOAD_RESULT',
+    payload: repos
+});
+
+export const getResult = searchTerm => {
+    return async dispatch => {
+        dispatch(loading(searchTerm));
+        try {
+            const repos = await fetchRepos(searchTerm);
+            console.log(repos)
+            dispatch(loadResult(repos))
+        } catch (err) {
+            console.warn(err.message);
+            dispatch({ type: 'SET_ERROR', payload: err.message })
+        };
+    };
+};
+export const fetchRepos = async(username) =>{
```

Comment:
Good use of a helper function, fetchRepos doesnt need to be exported though since its only used in the getResult function